src/components/Faqs.tsx

```diff
@@ -21,19 +22,42 @@ const FAQsPage: React.FC = () => {
   ];
 
   return (
-    <div className=" bg-white flex flex-col items-center mt-20">
-      <div className="w-full max-w-5xl p-8">
-        <h1 className="text-4xl font-medium text-primary my-10">FAQs</h1>
-        {faqs.map((faq, index) => (
-          <div key={index} className="mb-8">
-            <h2 className="text-xl font-medium text-secondary mb-2">{faq.question}</h2>
-            <p className="text-gray-500 text-sm font-normal mb-10 text-justify">{faq.answer}</p>
-            {index < faqs.length + 1 && <hr className="mt-6 border-light-grey" />}
+    <div className="bg-white flex flex-col items-center lg:flex-row lg:items-start px-4 md:px-8 lg:px-36">
+      {/* Image Section */}
+      <div className="text-center mb-8 w-full lg:w-auto">
+        <div id="features" className="bg-white py-12">
+          <div className="max-w-7xl mx-auto px-6 sm:px-8 lg:px-12">
+            <div className="text-start">
+              <h2 className="mt-4 text-3xl text-gray-600 tracking-wider text-primary font-">
+              FAQs
+              </h2>
+            </div>
           </div>
-        ))}
+          {/* Image display with responsive margin and padding */}
+          <div className="mt-8 lg:mt-14 flex justify-center lg:px-14 lg:pl-32">
```

Comment:
too much left padding here. you have lg:px-14 then you also have lg:pl-32. it means you have double padding to your left side. px means padding on x axis, which means left padding has a padding already from this point.

Reply:
Resolve issue